dotCMS/src/main/java/com/dotcms/auth/providers/jwt/beans/UserToken.java

```diff
@@ -186,5 +212,50 @@ public String getUserId() {
         return this.subject;
     }
 
+    public static final class Builder {
+        @JsonProperty(required = true)  private String id;
+        @JsonProperty(required = true)  private String subject;
+        @JsonProperty(required = true)  private String issuer = ClusterFactory.getClusterId();
+        @JsonProperty(required = true)  private Date   modificationDate = new Date();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field modificationDate has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)